bool reset
bool activate
bool open
bool close
bool release
uint8 request_pos
uint8 speed
uint8 force